features/gezag-minderjarige/1.4-is-uitspraak-gezag-aanwezig.feature

```diff
@@ -0,0 +1,150 @@
+      | naam                             | waarde                    |
+      | type                             | TweehoofdigOuderlijkGezag |
+      | minderjarige.burgerservicenummer | 000000036                 |
+      En heeft 'gezag' een 'ouder' met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000012 |
+      En heeft 'gezag' een 'ouder' met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000024 |
+
+#    Scenario: Voor het kind een uitspraak gedaan maar de indicatie betreft gezag is nog niet ingegaan
+#    Niet ondersteund, issue voor deze functionaliteit: https://github.com/BRP-API/brp-api-gezag/issues/171
+
+#    Scenario: Voor het kind een uitspraak gedaan met een indicatie voor gezag maar deze uitspraak is beëindigd
+#    Niet ondersteund, issue voor deze functionaliteit: https://github.com/BRP-API/brp-api-gezag/issues/171
+
+#    Scenario: Voor het kind een uitspraak gedaan met een indicatie voor gezag maar deze uitspraak is gecorrigeerd
+#    Niet ondersteund, issue voor deze functionaliteit: https://github.com/BRP-API/brp-api-gezag/issues/171
+
+  Regel: Uitspraak gezag is in onderzoek
+
+    Scenario: Voor het kind een uitspraak gedaan dat alleen ouder 1 gezag heeft, maar deze uitspraak staat op dit moment in onderzoek, resulteerd in GezagNietTeBepalen
+      Gegeven voor 'Babette' is een gerechtelijke uitspraak over het gezag gedaan met de volgende gegevens
+      | naam                                 | waarde                          |
+      | indicatie gezag minderjarige (32.10) | 1                               |
+      | ingangsdatum geldigheid (85.10)      | 20230101                        |
+      | aanduiding in onderzoek (83.10)      | 113210                          |
+      | datum ingang onderzoek (83.20)       | 20230101                        |
+      Als gezag wordt gezocht met de volgende parameters
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      En heeft de persoon een 'gezag' met de volgende gegevens
+      | naam                             | waarde                                                                                                                                      |
+      | type                             | GezagNietTeBepalen                                                                                                                          |
+      | toelichting                      | Gezag is niet te bepalen, omdat de volgende relevante gegevens in onderzoek staan. Persoonslijst van persoon: indicatie gezag minderjarige. |
+
+    Scenario: Voor het kind een uitspraak gedaan, deze uitspraak stond in onderzoek er is sprake van EenhoofdigOuderlijkGezag
```

Comment:
Vanuit functionele documentatie is deze scenario gelijk aan "Voor het kind een actuele uitspraak gedaan... " onder regel "Uitspraak gezag is aanwezig". Wel een goede test case.

Reply:
Op basis van nieuwe inzichten van de afgelopen week, wordt deze feature gezien als een implementatie feature. In dat opzicht kunnen deze scenario's blijven staan.